corehq/apps/locations/models.py

```diff
@@ -388,6 +407,19 @@ def linked_supply_point(self):
         from corehq.apps.commtrack.models import SupplyPointCase
         return SupplyPointCase.get_by_location(self)
 
+    @property
+    def location_type_object(self):
+        """
+        Brute force lookup for the LocationType object
+        that corresponds to this locations type.
+        This could definitely use a more efficient way,
+        but no domains at this point have a large list of
+        types.
+        """
+        for loc_type in Domain.get_by_name(self.domain).location_types:
+            if loc_type.name == self.location_type:
+                return loc_type
```

Comment:
This is copypasta'd from @twymer's other branch which is mired in QA.  Keeping this abstraction so we can easily swap it out once a better way to do this exists.